rules/BUILD.bazel

```diff
@@ -2,3 +2,7 @@ exports_files(
     glob(["*.bzl"]),
     visibility = ["//docs:__pkg__"],
 )
+
+exports_files([
+    "WorkspaceSettings.xcsettings",
```

Comment:
this file doesn't seem to exist in this package?

Reply:
will fix!